src/java.base/share/classes/java/lang/Integer.java

```diff
@@ -456,32 +457,6 @@ public static String toUnsignedString(int i) {
         return Long.toString(toUnsignedLong(i));
     }
 
-    /**
```

Comment:
Would better to update related comments as well

https://github.com/openjdk/jdk/blob/d2e11593006dc32fb8ebbaf12488b8758c8a19ee/src/hotspot/share/opto/stringopts.cpp#L1163-L1165

There are several occurrences in stringopt phase, grep to find them.